File: app/src/main/java/com/pomoremote/timer/TimerState.java
package com.pomoremote.timer;

public class TimerState {
    public static final String STATUS_STOPPED = "stopped";
    public static final String STATUS_RUNNING = "running";
    public static final String STATUS_PAUSED = "paused";

    public static final String PHASE_WORK = "work";
    public static final String PHASE_SHORT = "short";
    public static final String PHASE_LONG = "long";

    public String status;
    public String phase;
    public String next_phase;
    public double start_time;
    public double duration;
    public double remaining;
    public int completed;
    public String date;
    public long last_action_time;
    public int version;

    public TimerState() {
        this.status = STATUS_STOPPED;
        this.phase = PHASE_WORK;
    }
}

File: app/src/main/java/com/pomoremote/timer/TimerState.kt
package com.pomoremote.timer

class TimerState {
    companion object {
        const val STATUS_STOPPED = "stopped"
        const val STATUS_RUNNING = "running"
        const val STATUS_PAUSED = "paused"

        const val PHASE_WORK = "work"
        const val PHASE_SHORT = "short"
        const val PHASE_LONG = "long"
    }

    @JvmField var status: String = STATUS_STOPPED
    @JvmField var phase: String = PHASE_WORK
    @JvmField var next_phase: String? = null
    @JvmField var start_time: Double = 0.0
    @JvmField var duration: Double = 0.0
    @JvmField var remaining: Double = 0.0
    @JvmField var completed: Int = 0
    @JvmField var goal: Int = 8
    @JvmField var date: String? = null
    @JvmField var last_action_time: Long = 0
    @JvmField var version: Int = 2
}
